setup/quickstart/halyard-gce.md

```diff
@@ -0,0 +1,240 @@
+    --project=$GCP_PROJECT \
+    --display-name $GCS_SERVICE_ACCOUNT_NAME
+
+GCS_SERVICE_ACCOUNT_EMAIL=$(gcloud iam service-accounts list \
+    --project=$GCP_PROJECT \
+    --filter="displayName:$GCS_SERVICE_ACCOUNT_NAME" \
+    --format='value(email)')
+
+# permission to create/modify buckets in your project
+gcloud projects add-iam-policy-binding $GCP_PROJECT \
+    --role roles/storage.admin \
+    --member serviceAccount:$GCS_SERVICE_ACCOUNT_EMAIL
+```
+
+```bash
+GCP_PROJECT=$(gcloud info --format='value(config.project)')
+GCE_SERVICE_ACCOUNT_NAME=spinnaker-gce-account
+
+gcloud iam service-accounts create \
+    $GCE_SERVICE_ACCOUNT_NAME \
+    --display-name $GCE_SERVICE_ACCOUNT_NAME
+
+GCE_SERVICE_ACCOUNT_EMAIL=$(gcloud iam service-accounts list \
+    --filter="displayName:$GCE_SERVICE_ACCOUNT_NAME" \
+    --format='value(email)')
+
+# permission to create/modify instances in your project
+gcloud projects add-iam-policy-binding $GCP_PROJECT \
+    --member serviceAccount:$GCE_SERVICE_ACCOUNT_EMAIL \
+    --role roles/compute.instanceAdmin
+
+# permission to create/modify network settings in your project
+gcloud projects add-iam-policy-binding $GCP_PROJECT \
+    --member serviceAccount:$GCE_SERVICE_ACCOUNT_EMAIL \
+    --role roles/compute.networkAdmin
+
+# permission to create/modify firewall rules in your project
+gcloud projects add-iam-policy-binding $GCP_PROJECT \
+    --member serviceAccount:$GCE_SERVICE_ACCOUNT_EMAIL \
+    --role roles/compute.securityAdmin
+
+# permission to create/modify images & disks in your project
+gcloud projects add-iam-policy-binding $GCP_PROJECT \
+    --member serviceAccount:$GCE_SERVICE_ACCOUNT_EMAIL \
+    --role roles/compute.storageAdmin
+
+# permission to download service account keys in your project
+# this is needed by packer to bake GCE images remotely
+gcloud projects add-iam-policy-binding $GCP_PROJECT \
+    --member serviceAccount:$GCE_SERVICE_ACCOUNT_EMAIL \
+    --role roles/iam.serviceAccountActor
+```
+
+### Create Halyard host VM
+
+Create a VM with the service account:
+
+```bash
+HALYARD_HOST=$USER-halyard-`date +%m%d`
+
+gcloud compute instances create $HALYARD_HOST \
+    --project=$GCP_PROJECT \
+    --zone=us-central1-f \
+    --scopes=cloud-platform \
+    --service-account=$HALYARD_SERVICE_ACCOUNT_EMAIL \
+    --image-project=ubuntu-os-cloud \
+    --image-family=ubuntu-1404-lts \
+    --machine-type=n1-standard-4
+```
+
+SSH into the VM with the following ports forwarded as shown to allow access to
+the Spinnaker UI & API servers.
+
+> :warning: You need to SSH into the Halyard host VM from your local
+> workstation; SSHing from [Cloud Shell](https://cloud.google.com/shell/) or
+> another VM won't open the necessary SSH tunnels that will allow your local
+> web browser to access Spinnaker.
+
+```bash
+gcloud compute ssh $HALYARD_HOST \
+    --project=$GCP_PROJECT \
+    --zone=us-central1-f \
+    -- -L 9000:localhost:9000 -L 8084:localhost:8084
```

Comment:
This is the new arg passing style that gcloud is pushing us to, I think the `--ssh-flag` will be deprecated eventually